packages/account/src/Components/error-component/error-component.jsx

```diff
@@ -25,8 +25,8 @@ const ErrorComponent = ({ header, message, redirect_label, redirectOnClick, shou
                           refresh_message,
                       ]
             }
-            redirect_url={routes.trade}
-            redirect_label={redirect_label || <Localize i18n_default_text='Refresh' />}
+            redirect_urls={[routes.trade]}
+            redirect_labels={[redirect_label || <Localize i18n_default_text='Refresh' />]}
```

Comment:
nitpick suggestion: just to make it consistent with how other `redirect_labels` are handling `localize`

```suggestion
            redirect_labels={[redirect_label || localize('Return to Trade')]}
```